kevm-pyk/src/kevm_pyk/kproj/evm-semantics/serialization.md

```diff
@@ -349,6 +353,9 @@ Encoding
 
     rule #rlpEncodeTxData( DynamicFeeTxData(TN, TF, TM, TG, TT, TV, DATA, TC, [TA]) )
       => #rlpEncode( [ TC, TN, TF, TM, TG, #addrBytes(TT), TV, DATA, [TA] ] )
+   
+    rule #rlpEncodeTxData( BlobTxData(TN, TF, TM, TG, TT, TV, DATA, CID, [TA], TB, [TVH]) )
+      => #rlpEncode( [ CID, TN, TF, TM, TG, #addrBytesNotNil(TT), TV, DATA, [TA], TB, [TVH] ] )
```

Comment:
I'm wondering if it's worth casting `TT` to the `Account` sort and keep using `addrBytes`.
```suggestion
      => #rlpEncode( [ CID, TN, TF, TM, TG, #addrBytes({TT}:>Account), TV, DATA, [TA], TB, [TVH] ] )
```

Reply:
The idea here is to enforce that `TT` cannot be `.Account` due to this specification on the EPI:

>The field `to` deviates slightly from the semantics with the exception that it *MUST NOT* be `nil` and therefore must always represent a 20-byte address. This means that blob transactions cannot have the form of a create transaction.

Reply:
Yes. The reason I suggested this change is because `BlobTxData` defines `to` as `AccountNotNil` which is an alias for `Int`. So in this `#rlpEncode` rule, `TT` cannot be `.Account` to begin with. I think that casting it to the `Account` before inserting it in `#addrBytes` should be enough to avoid defining `#addrBytesNotNil`, but I might be wrong.

Reply:
Ok, that makes sense, thanks.